docs/dashboards/entity-dashboard.asciidoc

```diff
@@ -74,18 +67,26 @@ Interact with the table to filter data, view more details, and take action:
 * Click *View all* in the upper-right to display all user risk information on the Users page. 
 * Click the number link in the *Alerts* column to view the alerts on the Alerts page. Hover over the number and select *Investigate in timeline* (image:images/timeline-button-osquery.png[Investigate in timeline icon,20,20]) to launch Timeline with a query that includes the associated user name value.
 
-For more information about user risk scores, click the *Learn more* link in the table, or refer to <<user-risk-score>>. 
+For more information about user risk scores, refer to <<entity-risk-scoring>>. 
 
 [[entity-anomalies]]
 [float]
 == Anomalies
 
-Anomalies identify suspicious or irregular behavior patterns. The Anomalies table displays the total number of host and user anomalies identified by six predefined {ml} jobs (named in the Anomaly name column). These jobs must be installed and running to provide anomaly data.   
+Anomaly detection jobs identify suspicious or irregular behavior patterns. The Anomalies table displays the total number of host and user anomalies identified by these prebuilt {ml} jobs (named in the **Anomaly name** column).
+
+.Requirements
+[sidebar]
+-- 
+
+To display anomaly data, you must {ml-docs}/ml-ad-run-jobs.html[install and run] one or more <<prebuilt-ml-jobs, prebuilt anomaly detection jobs>>. You cannot add custom anomaly detection jobs to the Entity Analytics dashboard.
```

Comment:
Does it make sense to simply? 

`The Anomalies table displays the total number of host and user anomalies identified` to `The Anomalies table displays the total number of anomalies identified`